<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EhDbViewer</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="txtSearchBar"/>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_main">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="AspectRatioLabel" name="labelHoverPreview">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string>No preview</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Search History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="history_view"/>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter_result_column">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="TabbedSearchResult" name="tabSearchResult">
        <property name="elideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Metadata:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txtMetadataDisplay">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnListFullDatabase">
          <property name="text">
           <string>列出所有数据库条目</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTestEhRequest">
          <property name="text">
           <string>测试：Eh请求</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTestProgressDiag">
          <property name="text">
           <string>Test Progress Dialog</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbUseThumbnail">
          <property name="text">
           <string>use_thumbnail</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="imagePreviewLayoutWidget">
       <layout class="QVBoxLayout" name="imagePreviewLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Image Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="AspectRatioLabel" name="labelPreview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>1</height>
           </size>
          </property>
          <property name="text">
           <string>No image</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_config">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionImportFolder"/>
    <addaction name="actionImportEhViewerBackup"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menu_config"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportFolder">
   <property name="text">
    <string>Import Folder</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionImportEhViewerBackup">
   <property name="text">
    <string>Import from EhViewer Backup</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AspectRatioLabel</class>
   <extends>QLabel</extends>
   <header>widget/AspectRatioLabel.h</header>
  </customwidget>
  <customwidget>
   <class>TabbedSearchResult</class>
   <extends>QTabWidget</extends>
   <header>widget/TabbedSearchResult.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtSearchBar</tabstop>
  <tabstop>btnTestEhRequest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
